From b0bb600f7641078b6e748e67a9bd7c7a1accaa50 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dcermak@suse.com>
Date: Mon, 1 Mar 2021 08:33:59 +0100
Subject: [PATCH 12/12] Bump bcrypt_pbkdf version

There were no functional changes between 1.0.0 and 1.1.0:
https://github.com/net-ssh/bcrypt_pbkdf-ruby/compare/b5d1a921fcec8daa368236f5b4c469e08200fd5c...v1.1.0
so this update should be safe

(cherry picked from commit d996166098a22ab0212d31592a6e97fb50ea8a5e)
---
 vagrant.gemspec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vagrant.gemspec b/vagrant.gemspec
index edcdda02f..7c554d4fb 100644
--- a/vagrant.gemspec
+++ b/vagrant.gemspec
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version     = "~> 2.5", "< 2.8"
   s.required_rubygems_version = ">= 1.3.6"
 
-  s.add_dependency "bcrypt_pbkdf", "~> 1.0.0"
+  s.add_dependency "bcrypt_pbkdf", "~> 1.1"
   s.add_dependency "childprocess", "~> 4.0.0"
   s.add_dependency "ed25519", "~> 1.2.4"
   s.add_dependency "erubi"
-- 
2.30.1

